guilford_dps_offenders.yaml

```diff
@@ -0,0 +1,330 @@
+            target: ["RUDDY", "DARK", "FRECKLED", "LIGHT", "UNKNOWN", "ALBINO", "MEDIUM"]
+            goal: ["ruddy", "dark", "freckled", "light", "unknown", "albino", "medium"]
+      ol.haircolor:
+        type: "ol.haircolor"
+        # column: "OFFENDER HAIR COLOR CODE"
+        transforms:
+        - !<transforms.BooleanRegexTransform>
+          column: "OFFENDER HAIR COLOR CODE"
+          pattern: "(\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?)"
+          transformsIfTrue:
+          - !<transforms.ValueTransform>
+            value: null
+          transformsIfFalse:
+          - !<transforms.ColumnTransform>
+            column: "OFFENDER HAIR COLOR CODE"
+          - !<transforms.ReplaceTransform>
+            target: ["RED", "WHITE", "BLONDE", "BLACK", "SANDY", "OTHER", "UNKNOWN", "BROWN", "BALD", "GRAY"]
+            goal: ["red", "white", "blonde", "black", "sandy", "other", "unknown", "brown", "bald", "gray"]
+      ol.bodydescription:
+        type: "ol.bodydescription"
+        transforms:
+        - !<transforms.BooleanRegexTransform>
+          column: "OFFENDER BODY BUILD CODE"
+          pattern: "(\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?)"
+          transformsIfTrue:
+          - !<transforms.ValueTransform>
+            value: null
+          transformsIfFalse:
+          - !<transforms.ColumnTransform>
+            column: "OFFENDER BODY BUILD CODE"
+      ol.eyecolor:
+        type: "ol.eyecolor"
+        # column: "OFFENDER EYE COLOR CODE"
+        transforms:
+        - !<transforms.BooleanRegexTransform>
+          column: "OFFENDER EYE COLOR CODE"
+          pattern: "(\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?)"
+          transformsIfTrue:
+          - !<transforms.ValueTransform>
+            value: null
+          transformsIfFalse:
+          - !<transforms.ColumnTransform>
+            column: "OFFENDER EYE COLOR CODE"
+          - !<transforms.ReplaceTransform>
+            target: ["AMBER", "BLACK", "VIOLET", "HAZEL", "OTHER", "MIXED", "BROWN", "GRAY", "GREEN", "BLUE"]
+            goal: ["amber", "black", "violet", "hazel", "other", "mixed", "brown", "gray", "green", "blue"]
+    name: "offenderappearance"
+
+  offenderimmigration:
+    fqn: "ol.immigrationstatus"
+    entitySetName: "GuilfordImmigrationStatuses"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "OFFENDER NC DOC ID NUMBER"
+      ol.status:
+        type: "ol.status"
+        column: "OFFENDER CITIZENSHIP CODE"
+    conditions:
+    - !<conditions.BooleanRegexCondition>
+      column: "OFFENDER CITIZENSHIP CODE"
+      pattern: "(?i)(\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?)"
+      reverse: true
+    name: "offenderimmigration"
+
+  offendercommunication:
+    fqn: "ol.communication"
+    entitySetName: "GuilfordCommunication"
```

Comment:
Actually let's change this name to GuilfordLanguageSpoken so users know what's in it more clearly.

Reply:
done